src/screens/Server.js

```diff
@@ -199,20 +311,52 @@ class Server extends React.Component {
         <form onSubmit={e => { e.preventDefault(); }}>
           <div className="row mt-3">
             <div className="col-12">
-              <select onChange={this.handleSelectChange}>
+              { this.props.useWalletService && (
+                  <p className="input-label">{t`Base server`}:</p>
+                )
+              }
+              <select onChange={this.handleBaseURLSelectChange}>
                 <option value=""> -- </option>
                 {renderServerOptions()}
               </select>
             </div>
           </div>
+          {
+            this.props.useWalletService && (
+              <div className="row mt-3">
+                <div className="col-12">
+                  { this.props.useWalletService && (
```

Comment:
This line can only be rendered if `this.props.useWalletService` is true.
There should be no need to re-check it here.

Reply:
Done in 563884155ba01329272a1ca1e60bf3110223da28